### DISCUSSION TITLE
KafkaMirrorMaker2.0 with multiple replicas - not replicating the data

### DISCUSSION THREAD

**Discussion by lenin-joseph1** [strimzi/strimzi-kafka-operator#7784] [Q&A]
Hi, We are trying to run MM2(3.0.0) with 2 replicas, but this is causing an issue some times, where the worker gets created in one pod and tasks get created in other pod resulting in no data replication between clusters. 

The reason we would like to run with multiple replicas is to avoid an outage during AZ failure. So please let us know if this is possible.

**Comment by scholzj** [comment]
Sorry, but what does it mean that `the worker gets created in one pod and tasks get created in other pod`? There are only tasks what gets scheduled in Mirror Maker 2. The _workers_ are the pods themselves.
